### PR TITLE
DevWs for Most Target Limited containing containing ZigBee Switch Power

### PR DESCRIPTION
**Pull request by adurosmart**

### CODE REVIEW

**Comment by shrakuma** [comment]
@MAblewiczS @greens @tpmanley this is a FP submission. can you please review and target it to STG directly or master.

**Comment by shrakuma** [comment]
@MAblewiczS @greens please review these FP submission. should be straightforward

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 1]
```diff
@@ -46,6 +46,11 @@ metadata {
 
 		// Salus
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", manufacturer: "SALUS", model: "SX885ZB", deviceJoinName: "Salus miniSmartplug"
+        
+        //AduroSmart
+        fingerprint profileId: "0104", deviceId: "0051", inClusters: "0000, 0003, 0004, 0005, 0006, 0B04, 1000, 0702", outClusters: "0019", manufacturer: "AduroSmart Eria", model: "AD-SmartPlug3001", deviceJoinName: "Eria Zigbee Smart Plug"
```
Comment (MAblewicz) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: @adurosmart Could you change indentation to tabs instead of spaces?

**Review by MAblewicz — DISMISSED** [review]
@adurosmart Just change indentation to spaces and we are ready to go.

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy (hunk 1 above)
Comment (adurosmart) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy]: I already change indentation to tabs.
Thank you to review.

**Review by greens — APPROVED** [review]

**Pull request merged**